source/fundamentals/builders/projections.txt

```diff
@@ -11,3 +11,169 @@ Projections Builders
    :class: singlecol
 
 .. _projections-builders:
+
+Overview
+--------
+
+MongoDB supports **field projection**, specifying which fields to include and exclude when returning results from a
+query. Projection in MongoDB follows some basic rules:
+
+- the :guilabel:`_id` field is **always** included and must explicitly be excluded
+- specifying a field for inclusion implicitly excludes all other fields **except** the :guilabel:`_id` field
+- specifying a field for exclusion removes only that field in a query result
+
+Find more information :manual:`here </tutorial/project-fields-from-query-results/>`.
+
+
+The :java-core-api:`Projections <com/mongodb/client/model/Projections.html>` class provides static factory methods for
+all the MongoDB projection operators.  Each method returns an instance of the :ref:`Bson <bson>` type which you can pass
+to any method that expects a projection.
+
+.. tip::
+
+   For brevity, you may choose to import the methods of the ``Projections`` class statically:
+
+   .. code-block:: java
+
+      import static com.mongodb.client.model.Projections.*;
+
+   The examples below assume this static import.
+
+Inclusion
+---------
+
+Use the ``include`` method to specify the inclusion of one or more fields.
+
+The following example includes the :guilabel:`quantity` field and (implicitly) the :guilabel:`_id` field:
+
+.. code-block:: java
+
+   include("quantity")
+
+The following example includes the :guilabel:`quantity` and :guilabel:`totalAmount` fields and (implicitly) the
+:guilabel:`_id` field:
+
+.. code-block:: java
+
+   include("quantity", "totalAmount")
+
+Exclusion
+---------
+
+Use the ``exclude`` method to specify the exclusion of one or more fields.
+
+The following example excludes the :guilabel:`quantity` field:
+
+.. code-block:: java
+
+   exclude("quantity")
+
+The following example excludes the :guilabel:`quantity` and :guilabel:`totalAmount` fields:
+
+.. code-block:: java
+
+   exclude("quantity", "totalAmount")
+
+Exclusion of :guilabel:`_id`
+----------------------------
+
+Use the ``excludeId`` method to specify the exclusion of the :guilabel:`_id` field:
+
+.. code-block:: java
+
+   excludeId()
+
+.. note::
+
+   ``excludeId`` is a convenience method and is shorthand for:
+
+   .. code-block:: java
+
+      exclude("_id")
+
+Project an Array Element Match
+------------------------------
+
+Use the ``elemMatch`` methods to specify an array projection.
```

Comment:
This section is a bit confusing. ``elemMatch(String, Bson)`` returns the first element of an array that matches the Bson, but ``elemMatch(String)`` returns the first element that matches what? I think the tip below is intended to clarify this, and perhaps more seasoned java folks will understand, but I find this section needing a bit more clarity.

Reply:
Thanks. Like I said, I wasn't happy with it. I think the new version is a lot more clear. Thoughts?